src/v3/src/util/settingsUtils.ts

```diff
@@ -73,3 +73,25 @@ export const getLanguageCode = (widgetProps: WidgetProps): LanguageCode => {
 
   return (supportedLanguages[supportedPos] ?? config.defaultLanguage) as LanguageCode;
 };
+
+export const getBackToSignInUri = (widgetProps: WidgetProps): string | undefined => {
+  const { backToSignInLink, signOutLink } = widgetProps;
+  return backToSignInLink || signOutLink;
+};
+
+export const getBaseUrl = (widgetProps: WidgetProps): string | undefined => {
```

Comment:
For reference: https://github.com/okta/okta-signin-widget/blob/21b98876539d6c48a7108a44de337027085f41d7/src/models/Settings.ts#L393-L404